lib/routes/easynomad/joblist.ts

```diff
@@ -0,0 +1,56 @@
+import { Route, ViewType } from '@/types';
+import got from '@/utils/got';
+import MarkdownIt from 'markdown-it';
+const md = MarkdownIt({
+    html: true,
+    linkify: true,
+});
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/',
+    categories: ['popular'],
+    view: ViewType.Notifications,
+    example: '/easynomad',
+    radar: [
+        {
+            source: ['easynomad.cn'],
+        },
+    ],
+    name: '轻松游牧-远程工作聚集地',
+    maintainers: ['oldj'],
+    handler,
+
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+};
+
+async function handler() {
+    const host = `https://easynomad.cn`;
```

Comment:
```suggestion
    const host = 'https://easynomad.cn';
```
Only use template literals when necessary